crowdsec-docs/unversioned/console/remediation_metrics.mdx

```diff
@@ -0,0 +1,66 @@
+---
+id: remediation_metrics
+title: Remediation Metrics
+---
+
+:::info
+To receive remediation metrics, make sure you’re using **CrowdSec v1.6.3** or higher.  
+**Note:** Not all Remediation Components report metrics to the CrowdSec Console. For details, refer to the [official documentation](https://docs.crowdsec.net/u/bouncers/intro).
+:::
+
+## Introduction
+
+The **Remediation Metrics** page offers a clear and comprehensive view of the malicious activity that CrowdSec has detected and blocked on your infrastructure.  
+It provides key insights into:
+- The number and types of attacks
+- The impact of remediation measures
+- The estimated resources saved by stopping these threats
+
+The page is divided into three main sections:
+- **Malicious Intents** – A breakdown of attack types associated with IPs blocked by security engines and blocklists.
+- **Malicious Traffic Discarded** – Raw and estimated data showing how much malicious traffic has been dropped by your remediation components.
+- **Projected Resources Saved** – An estimate of the resources preserved thanks to traffic being blocked (e.g., storage, bandwidth, log volume).
+
+---
+
+## Malicious Intents
+
+At the top of the page, you'll see the **Total Prevented Attacks** for the selected time period. This gives you an immediate overview of how many threats CrowdSec has detected and blocked.
+
+![Total Prevented Attacks](/img/console/remediation_metrics/rc-metrics-total-prevented-attacks.png)
+
+The **Malicious Intents** section provides a detailed breakdown of the types of attacks that were prevented. These are based on the behavior and typology of IPs blocked by your remediation components, including blocklists and security engines.
+
+![Malicious Intents Breakdown](/img/console/remediation_metrics/rc-metrics-malicious-intents.png)
+
+---
+
+
+## Malicious Traffic Discarded
+
+This section highlights the amount of malicious traffic that has been blocked by your bouncers. It includes both raw and estimated data on discarded requests, packets, and bytes.
+
+- **Raw data** represents actual traffic dropped by your remediation components (bouncers), powered by blocklists and security engines.
+- **Estimated data** is calculated by applying a coefficient to the raw metrics to provide a projected view of saved resources.
```

Comment:
We need to provide calculation here or on console page to allow users to understands how we got from raw -> estimated.

Reply:
EG: For every packet we estimate that X requests could of been blocked.